Various small docs changes (#2753)
PX4/PX4-user_guide: en/config/airspeed.md

## Performing the Calibration

> The airspeed calibration needs to read a stable baseline with 0 airspeed in order to determine an offset.

Reviewer: Just not useful?

Author: That and that I think it can be miss-leading - it doesn't have to be exact 0, just to lower the better. 0 is quite impossible to achieve, there's always some noise on it.

   If they are swapped then the sensor will read a large negative differential pressure when you blow into the tube, and the calibration will abort with an error. 
   :::

> 1. The GUI of *QGroundControl* tells you if the calibration was successful or not. 

Reviewer: ```suggestion
1. *QGroundControl* then tells you if the calibration was successful or not. 
```

> 1. The GUI inside *QGroundControl* tells you if the calibration was successful or not. 

Reviewer 2: ```suggestion
1. The GUI of *QGroundControl* tells you if the calibration was successful or not. 
```

Author: Adapted in 9917fb34d4e30166ae4dac815dc01e5380115ae0
